spejstore/docker-compose.yml

```diff
@@ -0,0 +1,45 @@
+version: "3"
+services:
+  db:
+    image: ghcr.io/hackerspacekrk/spejstore/postgres-hstore:latest
+    build: postgres-hstore
+    restart: always
+    environment:
+      - POSTGRES_HOST_AUTH_METHOD=trust
+    volumes:
+      - spejstore-pg:/var/lib/postgresql/data
+  web:
+    image: ghcr.io/hackerspacekrk/spejstore:latest
+    restart: always
+    command: bash -c "python manage.py migrate && python manage.py runserver 0.0.0.0:8000"
+    secrets:
+      - spejstore_clientid
+      - spejstore_secret
+    environment:
+      SPEJSTORE_CLIENT_ID: $$(cat /run/secrets/spejstore_clientid)
+      SPEJSTORE_SECRET: $$(cat /run/secrets/spejstore_secret)
+    volumes:
+      - .:/code
+    ports:
+      - "8021:8000"
+    depends_on:
+      - db
+    env_file:
+      - spejstore.env
+    labels:
+      - "traefik.enable=true"
+      - "traefik.docker.network=system_default"
+      - "traefik.http.services.spejstore.loadbalancer.server.port=1880"
+      - "traefik.http.routers.spejstore.rule=Host(`spejstore.apps.hskrk.pl`)"
+      - "traefik.http.routers.spejstore.middlewares=authentik@docker"
+      - "traefik.http.routers.spejstore.tls=true"
+      - "traefik.http.routers.spejstore.tls.certresolver=myresolver"
+
+volumes:
+  spejstore-pg:
+
+secrets:
+  spejstore_secret:
```

Comment:
just be sure to add those secrets to swarm *BEFORE* you start everything

Reply:
To swarm = to portainer?